spec/rental_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec'
+require_relative '../rental'
+require_relative '../book'
+require_relative '../person'
+
+describe Rental do
+  let(:person) { Person.new(21, 'Moe', parent_permission: false) }
+  let(:rental) { Rental.new(date, book, person) }
+  let(:book) { Book.new('Title', ' Author') }
+  let(:date) { '2021-09-11' }
+
+  before do
+    rental
+  end
+
+  it 'initializes with date, book, person' do
+    rental = Rental.new(date, book, person)
+    expect(rental.date).to eq(date)
+    expect(rental.person).to eq(person)
+    expect(rental.book).to eq(book)
+  end
+  it 'can add rental to rentals' do
+    rental = Rental.new(date, book, person)
+    expect(book.rentals).to include(rental)
```

Comment:
- [ ] Great job so far ✨I noticed that the `Rental` class has an `add_rental` method but a test specifically for it is not present. It's a good practice to have tests for all the methods to ensure that they work as expected.. Kindly make sure to test it 💪

![image](https://github.com/H-Ramalan/school-library/assets/106119105/34a5f8aa-30f1-42ca-9eb7-2d77662d86aa)